src/transformation/mod.rs

```diff
@@ -40,4 +40,4 @@ mod to_trimesh;
 pub mod utils;
 
 #[cfg(feature = "wavefront")]
-pub mod wavefront;
+mod wavefront;
```

Comment:
This module doesn´t have to be public, as it's implementing functions directly on `TriMesh`, which is public.

Making it public warns about undocumented module, as well as having nothing to show in it (the added function is within `TriMesh`)